packages/dev/core/src/Misc/tools.ts

```diff
@@ -792,7 +792,21 @@ export class Tools {
         fileName?: string,
         quality?: number
     ): void {
-        if (successCallback) {
+        if (typeof fileName === "string") {

```

Comment:
This will fail if no filename and no callback was provided. It was working before. Unless I am missing something?

Reply:
You're right, I overlooked that due to not realizing that was an intended use case. I've fixed it and updated the documentation to more clearly describe that use case.